sdk/python/kfp/compiler/read_write_test.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 The Kubeflow Authors
```

Comment:
The file name doesn't follow our convention of `<source>_test.py`. Wonder if it might be slightly better to merge this into `compiler_test.py` potentially with the config file merged or sit separately? (Understand the compiler_test.py is already fairly large, so maybe this standalone test file is better, I don't have a strong opinion here, up to you).

Reply:
I'm partial to keeping this separate from `compiler_test.py` because of:
a) size --> as you say `compiler_test.py` is large and
b) test content --> `read_write_test.py` covers not only compilation (write), but also loading components as well (read). I think the distinctive naming is consistent with the distinctive test content.